### PR TITLE
[CBRD-25290] Added a test case to _db_auth's rows are not deleted when dropping an user

### PR DESCRIPTION
**Pull request by junsklee**
Refer to http://jira.cubrid.org/browse/CBRD-25290

### CODE REVIEW

File: sql/_13_issues/_24_1h/cases/cbrd_25290.sql [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+GRANT SELECT ON tbl to user_select;
+GRANT SELECT ON tbl to user_insert;
+GRANT SELECT ON tbl to user_update;
+GRANT SELECT ON tbl to user_delete;
+GRANT SELECT ON tbl to user_alter;
+GRANT SELECT ON tbl to user_index;
+GRANT SELECT ON tbl to user_execute;
+GRANT SELECT ON tbl to user_all_privileges;
+
```
Comment (kwonhoil) [inline sql/_13_issues/_24_1h/cases/cbrd_25290.sql:25]: GRANT SELECT ON tbl to user_select;
GRANT INSERT ON tbl to user_insert;
GRANT UPDATE ON tbl to user_update;
GRANT DELETE ON tbl to user_delete;
GRANT ALTER  ON tbl to user_alter;
GRANT INDEX  ON tbl to user_index;
GRANT EXECUTE ON tbl to user_execute;
GRANT ALL PRIVILEGES ON tbl to user_all_privileges;

File: sql/_13_issues/_24_1h/cases/cbrd_25290.sql [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+SELECT COUNT(*) FROM _db_auth WHERE grantee IS NULL;
+
+DROP USER user_select;
```
Comment (kwonhoil) [inline sql/_13_issues/_24_1h/cases/cbrd_25290.sql:28]: SELECT 'After deleting the user_select', COUNT(*) FROM _db_auth WHERE grantee IS NULL;

File: sql/_13_issues/_24_1h/cases/cbrd_25290.sql [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+DROP USER user_insert;
```
Comment (kwonhoil) [inline sql/_13_issues/_24_1h/cases/cbrd_25290.sql:31]: SELECT 'After deleting the user_insert', COUNT(*) FROM _db_auth WHERE grantee IS NULL;

File: sql/_13_issues/_24_1h/cases/cbrd_25290.sql [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+DROP USER user_update;
```
Comment (kwonhoil) [inline sql/_13_issues/_24_1h/cases/cbrd_25290.sql:34]: SELECT 'After deleting the user_update', COUNT(*) FROM _db_auth WHERE grantee IS NULL;

File: sql/_13_issues/_24_1h/cases/cbrd_25290.sql (hunk 1 above)
Comment (junsklee) [reply sql/_13_issues/_24_1h/cases/cbrd_25290.sql:25]: The wrong version had been commited; I have revised the file to the correct version in the latest commit.

Comment (junsklee) [reply sql/_13_issues/_24_1h/cases/cbrd_25290.sql:25]: An incorrect version of the file had been committed; I have revised the file to the correct version in the latest commit.

File: sql/_13_issues/_24_1h/cases/cbrd_25290.sql (hunk 3 above)
Comment (junsklee) [reply sql/_13_issues/_24_1h/cases/cbrd_25290.sql:31]: The suggestion has been implemented in the latest commit.

File: sql/_13_issues/_24_1h/cases/cbrd_25290.sql (hunk 2 above)
Comment (junsklee) [reply sql/_13_issues/_24_1h/cases/cbrd_25290.sql:28]: The suggestion has been implemented in the latest commit.

File: sql/_13_issues/_24_1h/cases/cbrd_25290.sql (hunk 4 above)
Comment (junsklee) [reply sql/_13_issues/_24_1h/cases/cbrd_25290.sql:34]: The suggestion has been implemented in the latest commit.

**Review by hgryoo — APPROVED** [review]

**Review by ssihil — APPROVED** [review]

**Review by bagus-kim — APPROVED** [review]

**Review by tw-kang — APPROVED** [review]

**Review by kwonhoil — APPROVED** [review]

**Pull request merged**